README.md

```diff
@@ -19,10 +19,9 @@ The goal of this specification is to enable the creation of interoperable tools
 
 ## Overview
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+This specification defines an OCI Image, consisting of a [manifest](manifest.md), a set of [filesystem layers](layer.md), and a [configuration](config.md).
 At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
```

Comment:
I'm fine with that change, but am not touching the line in this PR.  Maybe ask in #455 (which is touching the line)?